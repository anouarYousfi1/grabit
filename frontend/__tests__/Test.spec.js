/**
 * unit tests
 */

require("./components/home/Background.unit.test");
require("./components/home/ContentBrick.unit.test");
require("./components/home/Footer.unit.test");
require("./components/home/Header.unit.test");

/**
 * integration tests
 */

require("./components/home/Content.integration.test");
require("./components/home/HeaderContent.integration.test");

/**
 * e2e tests
 */

require("./components/home/Home.e2e.test");
