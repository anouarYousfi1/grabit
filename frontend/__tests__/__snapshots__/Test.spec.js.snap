// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content Brick component renders correctly 1`] = `ShallowWrapper {}`;

exports[`Content component renders correctly 1`] = `ReactWrapper {}`;

exports[`Footer component renders correctly 1`] = `ShallowWrapper {}`;

exports[`Header component renders correctly 1`] = `ShallowWrapper {}`;

exports[`HeaderContent component renders correctly 1`] = `ReactWrapper {}`;

exports[`Home Background component renders correctly 1`] = `ShallowWrapper {}`;
